tutorials/power_distribution/power_distribution.c

```diff
@@ -0,0 +1,26 @@
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+gpio_t DASHBOARD_POWER = PB0;
+gpio_t SERVICE_SECTION_POWER = PB1;
+gpio_t THROTTLE_POWER = PB2;
+
+void power_device(gpio_t pin) {
```

Comment:
I wouldn't call it `pin`. Mostly because they represent devices, so I would probably call it `device` or something like that.